featherpad/CMakeLists.txt

```diff
@@ -77,14 +77,25 @@ if(NOT WITHOUT_X11 AND UNIX AND NOT APPLE AND NOT HAIKU)
   endif()
   add_definitions(-DHAS_X11)
 else()
-  include_directories(${CMAKE_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}
-                      ${Qt5Core_INCLUDE_DIRS}
-                      ${Qt5Gui_INCLUDE_DIRS}
-                      ${Qt5Widgets_INCLUDE_DIRS}
-                      ${Qt5Svg_INCLUDE_DIRS}
-                      ${Qt5Network_INCLUDE_DIRS}
-                      ${Qt5PrintSupport_INCLUDE_DIRS}
-                      ${HUNSPELL_INCLUDE_DIRS})
+    if(QT_VERSION_MAJOR EQUAL 6)
```

Comment:
Oh, how did I miss these?! Perhaps because I was tired last night.

You could use `Qt${QT_VERSION_MAJOR}...` here (like `${Qt${QT_VERSION_MAJOR}Core_INCLUDE_DIRS}` and the same for others) because they're basically the same for Qt5 and Qt6. In other places, I separated them from each other because some include dirs weren't common.

And the same for the other block...

Would you please do it?